product_docs/docs/biganimal/release/getting_started/01_preparing_azure/index.mdx

```diff
@@ -0,0 +1,213 @@
+ ```
+
+1. Run the `azure-preflight.sh` script with the following options: 
+   - `<azure_region>` is the Azure region you plan to deploy your clusters.
+   - `<pg_type>` **e*N*s_v3** indicates the type of virtual machine instance using *N* cores.
+   - `<end_point>` indicates either **public** or **private** endpoints.
+   - `--ha` indicates that the instance is run with high availability enabled.
+   - `--with-infra` indicates that no clusters have been created in the region as yet.
+  
+  
+ For example, if you want to deploy a cluster in the `eastus2` region with `e2s_v3` cores, in a `public` network, and no existing cluster, run the following command:
+
+ ```
+ curl -sL https://raw.githubusercontent.com/EnterpriseDB/cloud-utilities/main/azure/azure-preflight.sh | bash -s -- --location eastus2 --pgtype e2s_v3 --endpoint public --with-infra
+ ```
+
+The script displays the following:  
+
+* A list of required Azure Service Providers and their registeration status. Ensure that you register the service providers that are displayed as *NotRegistered* in the *Registeration state* column. See [Register Azure service providers](#register-azure-service-providers)
+
+```
+ __OUTPUT__
+
+#######################
+# Provider            #
+#######################
+
+Namespace                                RegistrationPolicy    RegistrationState    ProviderAuthorizationConsentState
+---------------------------------------  --------------------  -------------------  -----------------------------------
+Microsoft.Capacity                       RegistrationRequired  NotRegistered
+Microsoft.ContainerInstance              RegistrationRequired  NotRegistered
+Microsoft.Compute                        RegistrationRequired  NotRegistered
+Microsoft.ContainerService               RegistrationRequired  NotRegistered
+Microsoft.KeyVault                       RegistrationRequired  NotRegistered
+Microsoft.ManagedIdentity                RegistrationRequired  NotRegistered
+Microsoft.Network                        RegistrationRequired  NotRegistered
+Microsoft.OperationalInsights            RegistrationRequired  NotRegistered
+Microsoft.OperationsManagement           RegistrationRequired  NotRegistered
+Microsoft.Portal                         RegistrationFree      Registered
+Microsoft.Storage                        RegistrationRequired  Registered
+Microsoft.AlertsManagement               RegistrationRequired  NotRegistered
+
+```
+ 
+ 
+* Whether you have sufficient limits on vCPUs for the `Standard_D2_v4` and `Standard_E2s_v3` VM size families in your region (and availability zone, if HA is enabled). Open a support request to remove SKU restrictions for the VM families with *NotAvailableForSubscription* displayed in the *Restrictions* column. See [Fix issues with SKU restrictions](#fix-issues-with-sku-restrictions)
```

Comment:
* Whether Azure can provide sufficient virtual machines for the `Standard_D2_v4` and `Standard_E2s_v3. Open a support request to remove SKU restrictions for the VM families with *NotAvailableForSubscription* displayed in the *Restrictions* column. See [Fix issues with SKU restrictions](#fix-issues-with-sku-restrictions)

Reply:
Done.